tests/factories/group.py

```diff
@@ -11,4 +11,4 @@ class Meta:
 
     name = 'eventgp1'
     user = factory.RelatedFactory(UserFactory)
-    user_id = 1
+    user_id = 2
```

Comment:
Why change this? This is propbably why it isn't working

Reply:
I tried previously with 1 and it wasn't working so set it to 2 seeing how it's done in speaker factory file. @iamareebjamal